templates/6_tma_batch_qc_metrics.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<p><code>extracted_imgs</code> makes it sound like this can be run immediately after extraction. However, this requires the renaming and reorganize notebook to have been run already. The names should make it obvious where this fits into the pipeline. Also some helpful text at the beginning of the notebook about when this notebook should be run, and what it's looking for.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>